intranet/apps/seniors/management/commands/cleanup_destinations.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+from ...models import Senior
+
+
+class Command(BaseCommand):
+    help = "Clean up senior destinations"
+
+    def handle(self, *args, **options):
+        if len(Senior.objects.all()) > 0:
+            yn = input("Are you sure you want to delete all senior destinations? [y/N]\n")
+            if yn == "Y" or yn == "y":
+                all_seniors = Senior.objects.all()
+                print(f"Deleting {len(all_seniors)} destinations.")
```

Comment:
```suggestion
                print(f"Deleting {all_seniors.count()} destinations.")
```

`.count()` returns the number of items in the QuerySet.